src/sidebar/util/test/copy-to-clipboard-test.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const { copyText } = require('../copy-to-clipboard');
+
+describe('copy-to-clipboard', () => {
+  beforeEach(() => {
+    sinon.stub(document, 'execCommand');
+  });
+
+  afterEach(() => {
+    document.execCommand.restore();
+  });
+
+  describe('copyText', () => {
+    /**
+     * Returns the temporary element used to hold text being copied.
+     */
+    function tempSpan() {
+      return document.querySelector('.copy-text');
+    }
+
+    it('copies the passed text to the clipboard', () => {
+      // We can't actually copy to the clipboard due to security restrictions,
+      // but we can verify that `execCommand("copy")` was called and that the
+      // passed text was selected at the time.
+      const testStr = `test string ${Math.random()}`;
+      document.execCommand.callsFake(() => {
+        assert.equal(document.getSelection().toString(), testStr);
```

Comment:
I had to look up `document.getSelection` because I forgot what it did; might be helpful to drop a comment here?

As I'm grokking this, the test can only go far as to create the temp span, stick the text in, and create a selection range. So this is verifying that the selection is correct, yes?

Reply:
> As I'm grokking this, the test can only go far as to create the temp span, stick the text in, and create a selection range. So this is verifying that the selection is correct, yes?

That is correct. I'm not sure I can add that much value in a comment here. Googling "document.getSelection" seems like the best way for a non-familiar reader to figure that out.